geopy/geocoders/base.py

```diff
@@ -45,6 +46,11 @@
 DEFAULT_TIMEOUT = 1
 DEFAULT_WKID = 4326
 
+
+def get_default_user_agent():
+    return "geopy/" + gu.get_version()
```

Comment:
Having it as a constant is cheaper. Not significant here, just the principle. You can mock anything, so, e.g., not tested: `with mock.patch('geopy.geocoders.base.DEFAULT_USER_AGENT', 'geopy/0.0.0')`. IIRC the usage below without a second parameter returns a MagickMock object, which is a callable instead of a string like you want.